src/components/ResultList/SimpleResultList.tsx

```diff
@@ -61,6 +92,7 @@ export const SimpleResultList = (props: ISimpleResultListProps): ReactElement =>
           highlights={highlights?.[index] ?? []}
           isFetchingHighlights={allowHighlight && isFetchingHighlights}
           useNormCite={useNormCite}
+          defaultCitation={defaultCitations?.get(doc.bibcode)}
```

Comment:
Where does the `.get` come from? Is that something useMemo is adding to the return?